### PR TITLE
todoman: install completions

### PR DESCRIPTION
**Pull request by triallax**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/todoman.rb [hunk 1]
```diff
@@ -75,6 +75,9 @@ class Todoman < Formula
 
   def install
     virtualenv_install_with_resources
+
+    bash_completion.install "contrib/completion/bash/_todo" => "todo"
+    zsh_completion.install "contrib/completion/zsh/_todo"
```
Comment (mhmdanas) [inline Formula/todoman.rb:80]: ZSH completion requires jq in certain circumstances, and due to a bug, can currently fail if it is not installed (see https://github.com/pimutils/todoman/issues/498). Should I do something about this?

Also, note that there are no Fish completions at the moment.

Comment (SMillerDev) [reply Formula/todoman.rb:80]: I'd suggest to add JQ as a build dependency then

Comment (carlocab) [reply Formula/todoman.rb:80]: Sounds like it should be a runtime dep

Comment (mhmdanas) [reply Formula/todoman.rb:80]: It would be an unnecessary dependency for people not using other shells, but that's fine I guess.

**Review by SMillerDev — APPROVED** [review]
Thanks @mhmdanas  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**